mozilla/fx-private-relay
# MPP-2200 Joint onboarding page

This PR fixes MPP-2200 

# New feature description

As a new subscriber to the Firefox Relay/VPN bundle I would like to be taken to a new welcome page that helps me onboard so I can get to both services easily and not get lost.

# Screenshot (if applicable)

<img width="1517" alt="image" src="https://user-images.githubusercontent.com/3924990/190308450-ef591f32-07d7-452e-a7a9-c971a931b070.png">

# How to test

Go to the page /vpn-relay-welcome with "bundle" flag on and after purchasing the vpn-relay product. We are using `has_vpn` just like we use `has_phone` and `has_premium`. If the user doesn't, we are redirecting them to the home page. I asked Tony if we wanted this page protected and he said he thought we should. Criteria says otherwise - comment left on MPP-2200. 

# Checklist
